static/assets/js/entity-reference/app.1.2.js

```diff
@@ -20,13 +20,13 @@
 
     function EntityReferenceCtrl($window, $http, $sce, $timeout, bbWait, localStorageService) {
         var self = this;
-        this.apiTitle = '';
         this.showErrorMessage = false;
+        this.swaggerCacheName = 'swaggerResponseCache-' + this.apiTitle;
 
         this.$onInit = onInit;
 
         function onInit() {
-            var swaggerResponseCache = localStorageService.get('swaggerResponseCache');
+            var swaggerResponseCache = localStorageService.get(self.swaggerCacheName);
```

Comment:
- I'll filter the apiTitle down to just letters and no other characters
 - I believe that the smallest limit is on mobile devices and is about 5mb. At least right now, we are no where near 5mb. Constit api's response is the largest right now and is about 314kb.